src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -14,7 +14,7 @@ public class EventCard extends UiPart<Region> {
     private static final String FXML = "EventListCard.fxml";
 
     /**
-     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * Remark: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
```

Comment:
Remark would make sense here too, so I'll leave it up to you to decide whether to change it here